### PR TITLE
feat:e2e- Fixed Home tests according to the UI changes

### PR DESCRIPTION
**Pull request by supriyagawade**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix
Fixed Home tests according to the UI changes

## Does this PR introduce a breaking change
No

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by deshmukhmayur — APPROVED** [review]

File: e2e/helper/apps.list.js [hunk 1]
```diff
@@ -37,22 +37,17 @@ export const microservicesreadmore = [
 export const builtapps = [
```
Comment (riginoommen) [inline e2e/helper/apps.list.js:37]:
```suggestion
 export const inBuiltApps = [
```

File: e2e/helper/apps.list.js [hunk 2]
```diff
@@ -37,22 +37,17 @@ export const microservicesreadmore = [
...
-    "/get-started/docs/faqs",
+    "/search",
     "/user-groups",
-    "/get-started/blog",
     "/get-started",
     "/components",
     "/"
 ];
 
 export const hostedapps = [
```
Comment (riginoommen) [inline e2e/helper/apps.list.js:47]:
```suggestion
 export const hostedApps = [
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

**Review by deshmukhmayur — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Comment by riginoommen** [comment]
@supriyagawade can you please rebase the branch and push it rest all changes looks good to me.

**Review by deshmukhmayur — APPROVED** [review]

**Comment by riginoommen** [comment]
@supriyagawade can you please squash the commits

**Comment by riginoommen** [comment]
@supriyagawade Please do fix the conflicts also

**Review by riginoommen — APPROVED** [review]

**Pull request merged**